### PR TITLE
Medium import

### PR DESCRIPTION
**Pull request by kentcdodds**

### CODE REVIEW

**Comment by kentcdodds** [comment]
Ok, so I just realized that some of these files are WAY big so I'm going to have them hosted elsewhere and just reference them with an absolute link. I'll also optimize them as well. I don't want these huge things in my git repo or on Netlify.

List them via: `find content/blog -type f -exec du -csh '{}' + | sort -h`

A 14M gif? No thanks!

Will be updating this soon.

File: content/blog/react-jsx-as-a-server-side-templating-language/index.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+  - React
+  - Jsx
+banner: ./images/banner.jpg
+bannerCredit:
+  'Photo by [Dwinanda Nurhanif
+  Mujito](https://unsplash.com/photos/3vp_7D1xd2A?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText)
+  on_[Unsplash](https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText)'
+---
+
+> _Another note:_ >
```
Comment (alaycock) [inline content/blog/react-jsx-as-a-server-side-templating-language/index.md:18]: It looks like the importer is inserting extra `>` characters in code blocks, and those characters are incorrectly being shown on the blog posts.

[Blog post on Netlify](https://5c71a54243826500078c1d34--kentcdodds.netlify.com/blog/react-jsx-as-a-server-side-templating-language)
[Blog post on Medium](https://blog.kentcdodds.com/react-jsx-as-a-server-side-templating-language-e238555dbc08)

Comment (kentcdodds) [reply content/blog/react-jsx-as-a-server-side-templating-language/index.md:18]: Thanks! There's a bunch of busted stuff like this that I'll probably fix retroactively 😬

**Comment by kentcdodds** [comment]
This was merged locally.

**Comment by kentcdodds** [comment]
@all-contributors please add @alaycock for content

**Pull request closed without merging**